app/src/main/java/org/greenstand/android/TreeTracker/activities/MainActivity.kt

```diff
@@ -53,8 +63,10 @@ class MainActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRes
 
         if (sharedPreferences.getBoolean(ValueHelper.FIRST_RUN, true)) {
 
-            if (sharedPreferences.getBoolean(ValueHelper.TREE_TRACKER_SETTINGS_USED, true)) {
-                sharedPreferences.edit()?.putBoolean(ValueHelper.TREE_TRACKER_SETTINGS_USED, true)?.apply()
+            if (sharedPreferences.getBoolean(
```

Comment:
These are manual formatting changes to adhere to the line length constraints set by ktlint. This is not auto-formatted and I have used my best judgment to make this readable while being within the line length limits.